### DISCUSSION TITLE
firmware "_tuf_"

### DISCUSSION THREAD

**Discussion by Maosoy** [gnuton/asuswrt-merlin.ng#661] [Q&A]
Hello!

Could you please tell me what is the difference between the firmware: 
“_TUF-AX5400_3004_388.8_2-gnuton1_tuf_puresqubi.w_” 
from
“_TUF-AX5400_3004_388.8_2-gnuton1_puresqubi.w_”
without the “*_*_tuf_*_*” designation?

I have a TUF-AX5400 router, which one is better to install on it?

**Comment by finkhi** [comment]
Can anyone respond to on this query?

**Comment by gnuton** [comment]
the difference is the theme. TUF has TUF UI theme the other one has just the classin heme.
The discussion was closed.
